integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -542,6 +543,50 @@ public static boolean verifyRollingRestartOccurred(Map<String, OffsetDateTime> p
     };
   }
 
+  /**
+   * Check the staus of the given server in domain status.
+   *
+   * @param domainUid uid of the domain
+   * @param domainNamespace namespace of the domain
+   * @param serverName name of the server
+   * @param podPhase phase of the server pod
+   * @param podReadyStatus status of the pod Ready condition
+   * @return true if the condition type has the expected status, false otherwise
+   */
+  public static Callable<Boolean> domainStatusServerStatusHasExpectedPodStatus(String domainUid,
+                                                                             String domainNamespace,
+                                                                             String serverName,
+                                                                             String podPhase,
+                                                                             String podReadyStatus) {
+    LoggingFacade logger = getLogger();
+
+    return () -> {
+      oracle.weblogic.domain.Domain domain =
+          assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace));
+
+      if (domain != null && domain.getStatus() != null) {
+        List<ServerStatus> serverStatusList = domain.getStatus().getServers();
+        logger.info(Yaml.dump(serverStatusList));
```

Comment:
(1) This, and the other logger.info should be changed to something like 'fine'/'finer'/'finest' I assume? Or even removed?
(2) If keeping them, I think they should have more context in their messages...

Reply:
Note that this is a test logger. This has followed other methods. It is helpful in debugging a test failure.

Reply:
Oops! I thought this was in the main code base.